src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -198,7 +198,7 @@ mkPragmaCompl label insertText =
     Nothing Nothing Nothing Nothing Nothing (Just insertText) (Just Snippet)
     Nothing Nothing Nothing Nothing Nothing
 
-cacheDataProducer :: HscEnv -> DynFlags -> TypecheckedModule -> [TypecheckedModule] -> IO CachedCompletions
+cacheDataProducer :: HscEnv -> DynFlags -> TypecheckedModule -> [ParsedModule] -> IO CachedCompletions
 cacheDataProducer packageState dflags tm tcs = do
```

Comment:
```suggestion
cacheDataProducer packageState dflags tm pmods = do
```
IIUC `tcs` was short for `typeCheckedModules`. In GHC's sources I see `pmod` as a short-cut for `parsedModule` in a couple places. There are a couple more instances of now misnamed `tcs`, `tms`, or `tcms` in this PR.

Reply:
I have taken a shot at fixing the names, but I'm not sure if it's what you had in mind. Rather than shortened type names, I've gone with more "semantic" names. Hope it works!

Reply:
Thanks, that's even better!